scripts/o3de/o3de/engine_template.py

```diff
@@ -2103,15 +2103,29 @@ def create_gem(gem_path: pathlib.Path,
 
     if summary:
         replacements.append(("${Summary}", summary))
+    else:
+        replacements.append(("${Summary}", ""))
```

Comment:
Alternatively you could use the [comprehension-if](https://docs.python.org/3/reference/expressions.html#conditional-expressions) syntax of python do do this in one line

```suggestion
    replacements.append(("${Summary}", summary if summary else ""))
```